newIDE/app/src/stories/index.js

```diff
@@ -113,6 +113,7 @@ import {
 } from '../Utils/GDevelopServices/ApiConfigs.js';
 import debuggerGameDataDump from '../fixtures/DebuggerGameDataDump.json';
 import profilerOutput from '../fixtures/ProfilerOutputsTestData.json';
+import consoleLogs from '../fixtures/ConsoleTestData.json';
```

Comment:
nitpicking: let's name this consoleTestData like the file.

Reply:
Should I also rename `profilerOutput` to `profilerOutputsTestData` too then?